cbw_api_toolbox/cbw_api.py

```diff
@@ -80,13 +82,23 @@ def _request(self, verb, payloads, body_params=None, params=None):
             body_params = json.dumps(body_params)
 
         try:
-            return requests.request(
-                verb,
-                route,
-                data=body_params,
-                params=params,
-                auth=CBWAuth(self.api_key, self.secret_key),
-                verify=self.verify_ssl)
+            if self.proxies != None:
```

Comment:
I thinks that request use kwargs, so the if / else a not mandatory.

Reply:
Yeah probably

Reply:
Thanks for the PR. Can you edit to remove the condition ? This line adds as well style issues : 
cbw_api_toolbox/cbw_api.py:85:12: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
cbw_api_toolbox/cbw_api.py:85:15: C0121: Comparison 'self.proxies != None' should be 'self.proxies is not None' (singleton-comparison)

Reply:
I've done it, is it good ? Or should i create a new PR ?